src/FileServiceProvider.php

```diff
@@ -22,17 +22,23 @@ function (Application $app) {
                 $awsCredentialsService = $app['aws-credentials.service'];
```

Comment:
This module makes an assumption about `$app['aws-credentials.service']` existing.

I think that should be pulled into this repo, otherwise this isn't reusable, right?